indico/modules/search/result_schemas.py

```diff
@@ -209,14 +216,23 @@ def _get_event_path(self, data):
 
 
 class BucketSchema(_ResultSchemaBase):
-    key = fields.String(required=True)
-    count = fields.Int(required=True)
-    filter = fields.String(required=True)
+    """Represents an individual aggregation bucket element"""
+
+    #: The aggregation key.
+    key: str = fields.String(required=True)
+    #: The number of elements.
+    count: int = fields.Int(required=True)
+    #: The key that identifies the element's filter.
+    filter: str = fields.String(required=True)
 
 
 class AggregationSchema(_ResultSchemaBase):
-    label = fields.String(required=True)
-    buckets = fields.List(fields.Nested(BucketSchema), required=True)
+    """Represents an aggregation list"""
```

Comment:
The title line of a docstring should end with a `.` (also in some other places like `BucketSchema`)